src/t-i18n.ts

```diff
@@ -1,47 +1,47 @@
-import { Replacements, Messages, MFunc, SetupOptions, ReactReplacements } from "./types";
+import { IcuReplacements, Messages, MFunc, SetupOptions, XmlReplacements, AnyReplacements } from "./types";
 import createCachedFormatter, { CachedFormatter, numberFormatOptions, dateTimeFormatOptions} from "./format";
-import { Plural, generator } from "./helpers";
-import parseICU from "./icu";
-import parseReact from "./react";
+import { Plural, generator, assign, splitReplacements } from "./helpers";
+import parseIcu from "./icu";
+import parseXml from "./xml";
 
 /**
  * T-i18n - lightweight localization
  * v0.3.1
```

Comment:
This should be bumped up to 0.4 (also in package.json) given the breaking changes.

Reply:
Yes, I wasn't sure if that was a change for this PR or if you wanted to do a version bump in its own PR.